### DISCUSSION TITLE
Set Seed values randomC and randomS in or-tools python

### DISCUSSION THREAD

**Discussion by avinashnavlani** [google/or-tools#3678] [Linear Solver questions] [answered]
Hi

I am trying to solve a complex MIP problem in OR tools using the CBC_MIXED_INTEGER_PROGRAMMING solver. I want to set the seed value RandomC or RandomS parameters mentioned in the thread https://github.com/coin-or/Cbc/discussions/379 for the deterministic and repeatable solutions. The ultimate goal is to resolve the non-deterministic behavior from my model output. 
Here are the initial 2 lines of the sample code.  Let me know where to set the seed values.

```py
from ortools.linear_solver import pywraplp

solver = pywraplp.Solver('simple_example', pywraplp.Solver.CBC_MIXED_INTEGER_PROGRAMMING)
```

**Comment by lperron** [comment, marked as answer]
answered on or-tools-discuss.
